upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -98,6 +100,9 @@ func (tf *TemplateFunctions) AddTo(dest template.FuncMap) {
 	dest["EncodeGCELabel"] = gce.EncodeGCELabel
 
 	dest["DnsControllerImage"] = tf.DnsControllerImage
+
+	// TODO: Only for AWS?
+	dest["AWSRegion"] = tf.AWSRegion
```

Comment:
Can we move this into apply_cluster.go and only set when we are in aws?  Or how does this run in GCE??